MusicTaste/MusicTaste/View Controllers/Login/LoginViewController.m

```diff
@@ -106,5 +102,22 @@ - (void)presentAlertControllerWithTitle:(NSString *)title
                          completion:nil];
     });
 }
+- (void)methodAWithCompletion:(void (^) (BOOL success))completion
+{
+    SpotifyAPIManager *api = [SpotifyAPIManager shared];
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, kNilOptions), ^{
+        [api setUpSpotifyWithCompletion:^(NSDictionary *data, NSError *error) {
+            if (!error) {
+            }else{
+            }
```

Comment:
Try not to commit code that's empty or not ready; you can save the code locally (let's talk about how), or maybe add a comment saying what you are planning to do there.